optaplanner-examples/src/test/java/org/optaplanner/examples/taskassigning/solver/TaskAssigningScoreConstraintTest.java

```diff
@@ -1,74 +0,0 @@
-/*
- * Copyright 2016 Red Hat, Inc. and/or its affiliates.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *      http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-package org.optaplanner.examples.taskassigning.solver;
-
-import java.util.Arrays;
-
-import org.junit.jupiter.api.Test;
-import org.optaplanner.core.api.solver.SolverFactory;
-import org.optaplanner.examples.taskassigning.app.TaskAssigningApp;
-import org.optaplanner.examples.taskassigning.domain.Customer;
-import org.optaplanner.examples.taskassigning.domain.Employee;
-import org.optaplanner.examples.taskassigning.domain.Priority;
-import org.optaplanner.examples.taskassigning.domain.Skill;
-import org.optaplanner.examples.taskassigning.domain.Task;
-import org.optaplanner.examples.taskassigning.domain.TaskAssigningSolution;
-import org.optaplanner.examples.taskassigning.domain.TaskOrEmployee;
-import org.optaplanner.examples.taskassigning.domain.TaskType;
-import org.optaplanner.test.impl.score.buildin.bendable.BendableScoreVerifier;
-
-public class TaskAssigningScoreConstraintTest {
```

Comment:
Why is this test going away?

Reply:
Honestly, I don't know. I'll see if I can bring it back.